napalm/nxos/nxos.py

```diff
@@ -1032,7 +1053,7 @@ def get_interfaces_ip(self):
                     'prefix_length': secondary_address_prefix
                 })
 
-        ipv6_command = 'show ipv6 interface'
+        ipv6_command = 'show ipv6 interface vrf all'
```

Comment:
Why is this changing? That probably should be in a separate change if it needs to change?

Reply:
I was adding the vrf support at that time, and yes it's not related to the bgp_neighbors